_posts/2015-12-09-foreman-community-newsletter-december-2015.md

```diff
@@ -0,0 +1,81 @@
+-   [foreman\_remote\_execution](https://github.com/theforeman/foreman_remote_execution)
+    0.1.1 was released November 12th.
+-   [foreman-xen](https://github.com/theforeman/foreman-xen)
+    0.2.3 (for Foreman >= 1.10) and 0.1.7 (for older versions) were released Novemeber 17th.
+-   [foreman-tasks](https://github.com/theforeman/foreman-tasks)
+    0.7.7 was released December 1st.
+
+### Other Updates
+
+-   A new [feature](https://groups.google.com/forum/#!topic/foreman-users/3RYfo0S5c8M) is planned to provide more information about smart proxies in Foreman, we would be happy to hear your input on this matter.
+-   An initial [Developer Handbook](http://theforeman.org/handbook.html) has been created to ease the entry of new contributers and help maintain high standards of existing ones.
+    Feel free to fork and open a pull request if you feel some important information is missing!
+-   The Foreman installer has been updated to drop puppet 2.7 support starting with the next version of Foreman (1.11).
+    For more information please read the [announcement](https://groups.google.com/forum/#!topic/foreman-users/YQNfxUzG_As).
+-   The development infrastructure has seen some updates - Redmine has been updated to version 2.6.8 and Jenkins has been updated to version 1.625.2.
+
+### New media and blogs
+
+-   The latest [community demo](https://www.youtube.com/watch?v=RCqwTtIXRSw) took place on November 19th. Highlights:
+    - yum repo export/import (beav) - https://www.youtu.be/RCqwTtIXRSw
+    - remote execution iteration 4 (stbenjam) - https://www.youtu.be/RCqwTtIXRSw?t=502
+    - foreman_ansible updates (dlobatog) - https://www.youtu.be/RCqwTtIXRSw?t=743
+    - scoped search for content hosts/subscriptions (jomitsch) - https://www.youtu.be/RCqwTtIXRSw?t=1501
+    - Rails 4 migration (the_doctor) - https://www.youtu.be/RCqwTtIXRSw?t=1689
+    - compliance reports for foreman_openscap (oprazak) - https://www.youtu.be/RCqwTtIXRSw?t=2201
+    - community announcements (gwmngilfen) - https://www.youtu.be/RCqwTtIXRSw?t=2359
+-   We held a deep dive about [Windows Imaging](https://www.youtube.com/watch?v=_R8-0KAkoPc) with Daniel Helgenberger. Thanks Daniel!
+-   We continuted our user case study series with Chris "discr33t" Pisano talking about his [High Availability](https://www.youtube.com/watch?v=bJ5a67SSA-s) setup with Foreman. Thanks Chris!
+-   Lukas Zapletal (lzap) gave a talk about Foreman at the Pilsen University. Read his report [here](https://groups.google.com/forum/#!topic/foreman-dev/l3-4a6pQTZU). Thanks Lukas!
+-   Daniel Lobato Garcia (dlobatog) gave a talk about [Rails security](https://speakerdeck.com/elobato/20-actionable-tips-to-secure-your-rails-application-rails-israel-15) at the Rails Israel conference. He also joined the Israeli Foreman team members to show off the project at the Red Hat booth.
+
+### Upcoming events
+
+-   This Thursday (Dec. 10th) is community demo time again.
```

Comment:
@GregSutcliffe I added this, but I couldn't find any announcement in the mailing lists.